src/components/panes/DonorPane/DonorPane.tsx

```diff
@@ -112,6 +119,40 @@ export function DonorPane() {
               />
               {isNameInvalid && <ErrorField text={paneTexts.nameError} />}
 
+              {IS_IN_COMPANY_MODE && (
+                <>
+                  <TextInput
+                    type="text"
+                    placeholder={paneTexts.companyNamePlaceholder}
+                    {...register('companyName', {
+                      required: !isAnonymous,
+                      minLength: 1,
+                    })}
+                    valid={!isCompanyNameInvalid}
+                  />
+
+                  {isCompanyNameInvalid && (
+                    <ErrorField text={paneTexts.companyNameError} />
+                  )}
+
+                  <TextInput
+                    type="text"
+                    placeholder={paneTexts.organizationNumberPlaceholder}
+                    {...register('organizationNumber', {
+                      required: !isAnonymous,
+                      validate: (val) =>
+                        val &&
+                        Validate.matches(val, /^(\d{1})(\d{5})\-(\d{4})$/),
```

Comment:
A regex I found on the web. Haven't tested extensively, but it lets my own org number through and it rejects some obviously wrong ones. Can add tests for it, if it's something we want to keep

Reply:
Seems to cover the most normal scenarios at least. Let's use it and see how it goes.